### PR TITLE
feat: show version on login page

### LINKED ISSUE #12909: site: display version in footer
a customer requested that we display the Coder version in the footer of the page, similar to how we did in Coder v1.

### PR DESCRIPTION
**Pull request by aslilac**
Screenshot 2024-04-22 at 11 47 44 AM

Closes #12909

### CODE REVIEW

File: site/src/modules/resources/AgentVersion.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const AgentVersion: FC<AgentVersionProps> = ({
   );
 
   if (status === agentVersionStatus.Updated) {
-    return <span>Updated</span>;
+    return null;
```
Comment (BrunoQuaresma) [inline site/src/modules/resources/AgentVersion.tsx:27]: Just checking if we want to do this since this looks to be outside of the scope.

**Review by BrunoQuaresma — APPROVED** [review]
LGTM! Nice work.

File: site/src/modules/resources/AgentVersion.tsx (hunk 1 above)
Comment (aslilac) [reply site/src/modules/resources/AgentVersion.tsx:27]: sneaking in out of scope things is my favorite 😝 but yes, this was intentional

**Comment by matifali** [comment]
The issue is how the version in the dashboard footer is on all pages all the time.
